README.md

```diff
@@ -3,3 +3,59 @@
 SwiftMemcache is a Swift Package in development that provides a convenient way to communicate with [Memcached](https://github.com/memcached/memcached) servers.
 
 ## Getting Started
+
+### Docker
```

Comment:
Can we move this to a `Contributing` section. This is not relevant for users of the package.